ledger-core/v2/virtual/testutils/stepchecker/checker.go

```diff
@@ -95,7 +92,7 @@ func getFuncName(step smachine.StateFunc) string {
 	return fullName
```

Comment:
please use convlog.GetStepName (from https://github.com/insolar/assured-ledger/pull/213) for similarity